src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Input validation purpose
```

Comment:
Incorrect / irrelevant header comment.
Please understand the purpose of header comments: other developers who extend your work should get necessary information about your classes and their methods.